tests/periph_flashpage/Kconfig

```diff
@@ -10,4 +10,5 @@ config APPLICATION
     default y
     imply MODULE_PERIPH_FLASHPAGE_RAW
     imply MODULE_PERIPH_FLASHPAGE_RWEE
+    select MODULE_OD
```

Comment:
use `CONFIG_MODULE_OD=y` in the application configuration file

Reply:
Ah but this is the application - should there be a separate config file?

Reply:
I mean to add it in `tests/periph_flashpage/app.config.test`, where the application enables the needed modules. Those modules are mandatory (as opposite to `periph_flashpage_raw` and `periph_flashpage_rwee` which are optional), so we define them in the configuration file, which is checked by `genconfig.py`.

Reply:
Ah I didn't realize we already have this file.